auth/permissions_cache.hh

```diff
@@ -30,7 +30,7 @@
 namespace std {
 
 inline std::ostream& operator<<(std::ostream& os, const pair<auth::role_or_anonymous, auth::resource>& p) {
-    os << "{role: " << p.first << ", resource: " << p.second << "}";
+    fmt::print(os, "{{role: {}, resource: {}}}", p.first, p.second);
```

Comment:
Minor: `auth/permissions_cache.hh` either implicitly includes `fmt/core.h` or does not include it at all. What do you think about including it explicitly?

Reply:
good point, lemme include it explicitly in the 2nd revision.